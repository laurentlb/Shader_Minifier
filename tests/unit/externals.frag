uniform int i, n, e;

int f() {
  int var1, var2, var3, var4, var5;
  return i;
}

int g() {
  int var1, var2, var3, var4, var5;
  return n;
}

int h() {
  int var1, var2, var3, var4, var5;
  return e;
}
