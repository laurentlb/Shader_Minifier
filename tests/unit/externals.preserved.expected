// Generated with  (https://github.com/laurentlb/Shader_Minifier/)
#ifndef EXTERNALS_PRESERVED_EXPECTED_
# define EXTERNALS_PRESERVED_EXPECTED_

const char *externals_frag =
 "uniform int i,n,e;"
 "int H()"
 "{"
   "return i;"
 "}"
 "int G()"
 "{"
   "return n;"
 "}"
 "int F()"
 "{"
   "return e;"
 "}";

#endif // EXTERNALS_PRESERVED_EXPECTED_
